src/main/java/com/datastax/fallout/components/common/provisioner/SshOnlyProvisioner.java

```diff
@@ -126,7 +126,7 @@ protected CheckResourcesResult reserveImpl(NodeGroup nodeGroup)
     protected boolean startImpl(NodeGroup nodeGroup)
     {
         return nodeGroup.waitForAllNodes(node -> {
-            String userName = FalloutPropertySpecs.userPropertySpec.value(node);
```

Comment:
isnt `FalloutPropertySpecs.userPropertySpec` now unused in the whole codebase?
thats why i thought we should remove it (i.e. also as an alias of the property we use here now).